src/test/java/com/ays/AbstractRestControllerTest.java

```diff
@@ -37,6 +38,7 @@ public abstract class AbstractRestControllerTest extends AbstractTestContainerCo
     protected MockMvc mockMvc;
     protected AysToken mockAdminUserToken;
     protected AysToken mockUserToken;
+    protected AysToken mockSuperAdminToken;
```

Comment:
Bu kısmı daha okunaklı olması için aşağıdaki gibi sıralayabilir miyiz?

```suggestion
    protected MockMvc mockMvc;
    protected AysToken mockSuperAdminToken;
    protected AysToken mockAdminUserToken;
    protected AysToken mockUserToken;
```

Reply:
Bu kısmı revise ettim